### PR TITLE
Cleanup comments and docs

### PR DESCRIPTION
**Pull request by jngrad**
Part of the [release checklist](https://github.com/espressomd/espresso/wiki/release-checklist#for-all-releases): spell and grammar check comments and docs.

### CODE REVIEW

File: doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb [hunk 1]
```diff
@@ -286,7 +286,7 @@
    "source": [
     "Note that we declared a <tt>lj_cut</tt>. This will be used as the cut-off radius of the Lennard-Jones potential to obtain a purely repulsive WCA potential.\n",
     "\n",
-    "Now we setting up the system. The length of the simulation box is calculated using the desired area fraction and the area all particles occupy. Then we create the **ESPResSo** system and pass the simulation step. For the Verlet list skin parameter we use the build-in tuning algorithm of **ESPResSo**. After that we setting up the thermostat which is, in our case, a Langevin thermostat to simulate in an NVT ensemble. \n",
+    "Now we setting up the system. The length of the simulation box is calculated using the desired area fraction and the area all particles occupy. Then we create the **ESPResSo** system and pass the simulation step. For the Verlet list skin parameter we use the built-in tuning algorithm of **ESPResSo**. After that we set up the thermostat which is, in our case, a Langevin thermostat to simulate in an NVT ensemble. \n",
```
Comment (pkreissl) [inline doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb]: Now we **set** up [...]

File: src/core/bonded_interactions/angle_common.hpp [hunk 2]
```diff
@@ -78,7 +78,7 @@ calc_vectors_and_cosine(Utils::Vector3d const &r_mid,
  *  @param[in]  r_right          Position of third/right particle.
  *  @param[in]  forceFactor      Angle force term.
  *  @param[in]  sanitize_cosine  Sanitize the cosine of the angle.
- *  tparam      ForceFactor      Function evaluating the angle force term
+ *  @tparam      ForceFactor     Function evaluating the angle force term
```
Comment (pkreissl) [inline src/core/bonded_interactions/angle_common.hpp]: The space should have been removed before `ForceFactor` to compensate for the added `@`.
That is, remove one space before `ForceFactor` and (re)add one after.

**Review by pkreissl — CHANGES_REQUESTED** [review]

File: doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb (hunk 1 above)
Comment (jngrad) [reply doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb]: Thanks! Don't know how I missed that.

File: src/core/bonded_interactions/angle_common.hpp (hunk 2 above)
Comment (jngrad) [reply src/core/bonded_interactions/angle_common.hpp]: Fixed.

**Review by pkreissl — APPROVED** [review]

**Review by RudolfWeeber — APPROVED** [review]
Based on @pkreissl's review

**Comment by jngrad** [comment]
bors r=RudolfWeeber

**Pull request merged**